references/cicd-sharedflow-pipeline/test/integration/apiproxy/Test-SF-Security-v1.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ Copyright 2020 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<APIProxy revision="1" name="Test-SF-Security-v1">
+    <ConfigurationVersion minorVersion="0" majorVersion="4"/>
```

Comment:
config version can be removed

Reply:
Same as sample in [cicd-pipeline](https://github.com/apigee/devrel/blob/c8191f08730c93adb1516d9b0eac313668fbc6d9/references/cicd-pipeline/apiproxy/airports-cicd-v1.xml#L19)